### ISSUE TITLE
TalkBack doesn't work properly

### ISSUE THREAD

**Issue by ESurina** [AdguardTeam/AdGuardVPNForAndroid#388]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard VPN version

2.6

### Operating mode

None

### Environment

```markdown
- OS: Android 12
- Device: Xiaomi
- Firmware:
```

### Root access

- [ ] Yes, I have it.

### Issue Details

Steps to reproduce:
1. Turn on TalkBack in the system settings

Not voiced:
- Reload button on the list of locations
- Cross icons in the text fields
- Eye icon in the password field

Voiced incorrect:
- Exclusions on the list (for example, zero x zero Epic games)
- A number of exclusions (for example, there is 1 exclusion on the list, but it says "2 of 2")
- Numbering of DNS servers (the heading is voiced like "2 of 17")
- About screen (the heading is voiced like "About, double tap to activate")
- Exclusion (website) details screen (the heading is voiced like "2 of 5")

### Expected Behavior

Everything is voiced properly

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>
